.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,15 @@
-#### Code Sample, a copy-pastable example if possible
+#### A small, complete example of the issue
```

Comment:
I think you also need ``#`` only (and not multiples), and this should be inside the quoting